app/tests/unit/components/validators.spec.js

```diff
@@ -0,0 +1,1476 @@
+const { validators, validatorUtils }= require('../../../src/components/validators');
+const {realSmallFile, smallFile} = require('./base64Files');
+
+describe('validatorUtils.isEmail', () => {
```

Comment:
As a side note, you are allowed to nest multiple describe clauses together if they better group things together for test visual representation